.github/ISSUE_TEMPLATE.md

```diff
@@ -8,11 +8,11 @@ Prefer live chat? Message our engineers on our Gitter channel at https://gitter.
 
 **FEATURE REQUEST**
 
-1. Please describe the feature you are requesting.
+1. Does an issue already exist addressing this request? If yes, please react to the existing issue with :+1:. If not, move on to step 2.
```

Comment:
> If yes, please react to the existing issue with 

nit: to be even clearer, I'd spell this "please add a 👍  reaction to the existing issue" because that exactly matches Github's terminology, and our users may not be native english speakers.